### DISCUSSION TITLE
Pressing tab on hitobject sound edit panel doesn't switch text fields (annoying bug?)

### DISCUSSION THREAD

**Discussion by Paladynee** [ppy/osu#17111] [Ideas] [answered]
### Its annoying to write hitobject bank name, go over to my mouse, click on the volume text field, write volume level WHILE I could've just done bank name > tab > volume. Its minor, but it adds up quickly.

**On the beatmap editor**, click on a **hitobject**'s _sound edit panel_ (_below the note on the timeline_), write something in the **Bank Name** to make sure you're focused on it, then press tab. 

**What should've happened:** text field focus changed to Volume
**What actually happened:** no act 

Proof: https://streamable.com/yz8cpu

**Comment by bdach** [comment, marked as answer]
will be fixed by #17116
